addons/account/static/src/js/tours/account.js

```diff
@@ -115,14 +120,14 @@ registry.category("web_tour.tours").add('account_tour', {
         run: "click",
     },
     {
-        isActive: ["auto"],
-        trigger: "div.modal-dialog",
-    },
-    {
-        trigger: ".modal button[name=document_layout_save]",
+        trigger: ".modal button[name=document_layout_save]:contains(continue)",
         content: _t("Configure document layout."),
         run: "click",
     },
+    {
+        isActive: ["auto"],
+        trigger: "body:not(:has(.modal:contains(configure your document layout)))",
```

Comment:
Isn't this very fragile because of the user string?
Does it behave nicely in onboardings in other languages?
Is it even necessary to have the user string in there?

Reply:
My bad, this PR is a draft